docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -1,70 +1,58 @@
 # Apama Software Management Plugin
 
-The apama plugin can be used to install and manage apama artefacts.
-This plugin can be used to install an apama project on a device as well as inject monitor files into an already installed project.
+The Apama plugin, part of the Apama thin-edge support package, can be used to install Apama projects using the Cumulocity software management feature.
 
+## Add the Apama thin-edge support package to the tenant software repository and deploy it to thin-edge
```

Comment:
You might also wanna provide a link to where the user can download this apama thin-edge support package. I also hope that this support package will work even on a machine where apama is already installed, in which case, it will just install the Apama thin-edge plugin only.

Reply:
There's not a fixed URL for the package - it depends on the version. The install instructions should be to install the apama-repo package (for which there is a fixed URL) and then install it as here using apt from the repo

Reply:
c622e75